docker/handshake-api/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:14.5
+
+WORKDIR /usr/app
+
+RUN yarn init -y
+RUN yarn add body-parser express hs-client
```

Comment:
These two should be run together, there is no need to create two layers.
`yarn init -y && yarn add body-parser express hs-client`

Reply:
Also... are you sure we should skip `package.json`? We can but we won't have the peace of mind that we're running the same code each time - if a new major version of one of the libraries comes out we'll automatically get it and it might not work with the rest and all that stuff.

It would e safer to have `package.json` and `package-lock.json` and use `yarn install --frozen-lockfile`, so we really know what we're running.

Reply:
I will fill that gap when I migrate this to monorepo, it's temporary for now.

Reply:
OK, we'll leave it as a F/U then.